`stainless` is broken with `rustc 1.14.0-nightly (f09420685 2016-10-20)`

`stainless` seems to break with `rustc 1.14.0-nightly (86affcdf6 2016-09-28)` with the following error

```rust
error[E0308]: mismatched types
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.7/src/describe.rs:68:96
   |
68 |     let state: DescribeState = Parse::parse(&mut parse::tts_to_parser(cx.parse_sess(), tokens, cx.cfg()), (sp, &mut*cx, Some(name)));
   |                                                                                                ^^^^^^^^ expected struct `std::vec::Vec`, found reference
   |
   = note: expected type `std::vec::Vec<syntax::ptr::P<syntax::codemap::Spanned<syntax::ast::MetaItemKind>>>`
   = note:    found type `&std::vec::Vec<syntax::ptr::P<syntax::codemap::Spanned<syntax::ast::MetaItemKind>>>`

error: aborting due to previous error
```

The last working version of stainless I checked was `rustc 1.14.0-nightly (86affcdf6 2016-09-28)`

--- a/src/describe.rs
+++ b/src/describe.rs
@@ -65,7 +65,7 @@ pub enum SubBlock {
 pub fn describe<'a>(cx: &'a mut base::ExtCtxt, sp: codemap::Span,
                 name: ast::Ident, tokens: Vec<tokenstream::TokenTree>) -> Box<base::MacResult + 'a> {
     // Parse a full DescribeState from the input, emitting errors if used incorrectly.
-    let state: DescribeState = Parse::parse(&mut parse::tts_to_parser(cx.parse_sess(), tokens, cx.cfg()), (sp, &mut*cx, Some(name)));
+    let state: DescribeState = Parse::parse(&mut parse::tts_to_parser(cx.parse_sess(), tokens, cx.cfg().clone()), (sp, &mut*cx, Some(name)));
 
     // Export the new module.
     base::MacEager::items(SmallVector::one(state.generate(sp, cx, None)))

